firmware/projects/EV5/debug/IoCheckoutFc/config.yaml

```diff
@@ -3,4 +3,4 @@ canGen:
   busses:
     - busName: io
       dbcFiles:
-        - "io.dbc"
+        - "../../io.dbc"
```

Comment:
io.dbc should be moved into the project directory, so reference it with just `"io.dbc"`